src/components/Navbar.jsx

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'react-router-dom';
+
+export default function Navbar() {
```

Comment:
- Optional
Consider using the arrow function instead of using the function declaration way. It is better to be consistent with our code base. Since you have used arrow functions throughout your application, it would make much more sense if you use the arrow function here. And also, we should be using more ES6 features now.

Reply:
Sorry, I forgot to use the arrow function here Thanks for your suggestion I will fix it. 😊